src/lib/utils.js

```diff
@@ -488,6 +488,24 @@ const validateActorName = (actorName) => {
     }
 };
 
+/**
+ * Checks whether a Makefile target exists in an actor directory, or not
+ *
+ * @param {string} actorDirectory Path to the actor's root directory
+ * @param {string} target The Makefile target for which to chck
+ * @return {bool} True if the specified target exists in the Makefile in the actor directory, otherwise false
+ */
+const checkIfMakefileTargetExists = (actorDirectory, target) => {
+    const makefilePath = path.join(actorDirectory, 'Makefile');
+    if (fs.existsSync(makefilePath)) {
+        const makefileContents = fs.readFileSync(makefilePath, { encoding: 'utf-8' });
```

Comment:
I wonder if 'utf-8' can be assumed

Reply:
Hm, good point, but this would become way harder if we had to detect encodings. 

I think at this point everybody should be using `utf-8` unless in some very special situations, so I would keep it like this, and make it more complicated only if needed.